projects/sbb-esta/angular-business/status/src/status/status.component.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../../../angular-core/styles/common/variables';
+@import '../../../../angular-core/styles/common/colors';
+
+:host {
+  display: flex;
+  flex-direction: row;
+
+  font-family: $fontSbbRoman;
+  font-size: 0.9375rem;
```

Comment:
Feel free to use the pxToRem function. This way we can use the values provided by the design specification (which are usually in pixel), while also providing rem values for consumers.